slobs_CI/install-script-win.cmd

```diff
@@ -1,4 +1,4 @@
-set DEPS=dependencies2017.2
+set DEPS=dependencies2017.3
```

Comment:
What did we update in our dependencies?

Reply:
It comes from https://github.com/stream-labs/obs-studio/pull/206 

My branch include this change as it initially started from streamlabs-25.0.8 branch.

Reply:
Sounds good, seems like this change has never been merged into the base branch which is the current preview.